app/src/test/java/org/simple/clinic/summary/updatephone/UpdatePhoneNumberDialogControllerTest.kt

```diff
@@ -85,16 +90,20 @@ class UpdatePhoneNumberDialogControllerTest {
       validationError: Result
   ) {
     val newNumber = "123"
-    val existingPhoneNumber = TestData.patientPhoneNumber(patientUuid = patientUuid, number = "old-number")
+    val existingPhoneNumber = TestData.patientPhoneNumber(
+        uuid = UUID.fromString("ab3f84b5-683f-49ae-987a-e319fd1db7d2"),
+        patientUuid = patientUuid,
+        number = "old-number"
```

Comment:
This is a nitpick, but while we are here, can we also update the data to be more "realistic" like using something like "1234567890" instead of an alphabetical string?